kalite/student_testing/tests.py

```diff
@@ -17,400 +17,401 @@
 
 logging = settings.LOG
 
+if "Nalanda" in settings.CONFIG_PACKAGE:
```

Comment:
Can you use `unittest.skipIf` instead? That would be cleaner than putting a toplevel `if` statement here.